guides/security/common_attack_vectors.md

```diff
@@ -0,0 +1,69 @@
+# Common attack vectors
+
+There are multiple ways a web application can be exploited. These are some of the most common attack vectors your application should be protected from:
+
+## Protect against Cross-Site Scripting (XSS)
+
+Cross-Site Scripting, most commonly known as XSS, is one of the most prominent attack methods on web applications.
+
+It consists of injecting an arbitrary script on a page. When another user is browsing the page, it will execute the script. The script could do anything: modify the user cookies, trigger a request on the user's behalf, copy some sensitive data and send it elsewhere, redirect the user to another site, etc.
+
+Any element on a page that renders user input data is susceptible to being vulnerable to XSS attacks.
+
```

Comment:
```suggestion
Even SVG files can be a vector for XSS if not handled carefully. While using an SVG as an `<img src="...">` is generally safe (scripts inside won't execute), injecting raw SVG content into the DOM via `innerHTML` or `dangerouslySetInnerHTML` can execute embedded scripts or malicious elements like `<script>`, `<image onerror>`, or `<foreignObject>`. Always sanitize SVG content before injecting it into the DOM.
```

I think it would be useful to add this note so we keep it in mind, especially when allowing clients to upload SVG files.